Library/Homebrew/formula_installer.rb

```diff
@@ -802,6 +807,8 @@ def finish
       post_install
     end
 
+    dsymutil(keg) if debug_symbols?
```

Comment:
This method name could be improved, it's not obvious what this does.

Reply:
Done in e46a61e181f8e324d1bc4222bedf692c11ad84fa